product_docs/docs/pgd/5/terminology.mdx

```diff
@@ -2,83 +2,120 @@
 
 #### CAMO or commit-at-most-once
 
-Wraps Eager Replication with additional transaction management at the application level to guard against a transaction being executed more than once. This transaction management is critical for high-value transactions found in payments solutions. It's roughly equivalent to the Oracle feature Transaction Guard.
+High value transactions in some applications require that the application is able to not only confirm that the transaction has been committed but that the transaction is only committed once or not at all. To ensure this happens in PGD, CAMO can be enabled allowing the application to actively participate in the transaction. 
+
+#### Conflicts
+
+As data is replicated across the nodes of a PGD cluster, there may be occasions when changes from one source clash with changes from another source. This is a conflict and can be handled with conflict resolution (rules which decide which source is correct or preferred), or avoided with conflict-free data types.
+
+#### Consensus
+
+How [Raft](#raft) makes group-wide decisions. Given a number of nodes in a group, Raft looks for a consensus of the number of nodes/2+1 voting for a decision. For example, when a write leader is being selected, a Raft consensus is sought over which node in the group will be the write leader. Consensus can only be reached if there is a quorum of voting members.
+
+#### Cluster
+
+Generically, a cluster is a group of multiple redundant systems arranged to appear to end users as one system. See also [PGD clusters](#pgd-clusters), [Kubernetes clusters](#kubernetes-clusters) and [Postgres clusters](#postgres-cluster).
 
-#### Clustering
+#### DDL
 
-An approach for high availability in which multiple redundant systems are managed to avoid single points of failure. It appears to the end user as one system.
+Data Definition Language - The subset of SQL commands that deal with the defining and managing the structure of a database. DDL statements can create, modify and delete objects - schemas, tables and indexes - within the database. Common DDL commands are CREATE, ALTER and DROP.
 
-#### Data sharding
+#### DML
 
-Enables scaling out a database by breaking up data into chunks called *shards* and distributing them across separate nodes.
+Data Manipulation Language - The subset of SQL commands that deal with manipulating the data held within a database. DML statements can create, modify and delete rows within tables in the database. Common DML commands are INSERT, UPDATE and DELETE. SELECT is also often referred to as DML, although it is actually part of the 
 
-#### Eager Replication for PGD
 
-Conflict-free replication with all cluster members. Technically, this is synchronous logical replication using two-phase commit (2PC).
+#### Eager
+
+Eager is a synchronous commit mode which avoids conflicts by detecting incoming potentially conflicting transactions and “eagerly” one of aborting them to maintain consistency. 
```

Comment:
@djw-m should we define how a commit mode and commit scope are different? I imagine something very simple i.e. commit mode is a setting on a commit scope, see eager etc.?

Reply:
No, because commit scopes are rules and things like eager are modes applied to the things that are applied to various parts of the transactions in scope.

Reply:
As part of the Postgres-R project, I had lengthy discussions with people claiming it clearly was eager and async. "Eager is a synchronous commit mode" used to be my original stance, from which I (personally) have diverged since.

However, for all PGD purposes, it is okay to state that.

Reply:
I think the query is more down to the fact that we refer to commit scopes as both the rules and the mechanisms enabled within the commit scopes (GC, LC, Eager etc)....

Going forward, I want to refer to commit scopes only as the rules in PGD that scope and set the commit modes which control how commits take place.

Commit kind and commit scope kind I've seen in other docs, but the former feels overly singular and the latter just feels like it's doubling down on the confusing use of commit scope.

Reply:
WFM